src/types/Widgets.ts

```diff
@@ -1,6 +1,10 @@
 import { JSONSchema7TypeName } from 'json-schema';
+import type { WidgetProps } from '@rjsf/core';
+import { ChangeEventHandler } from 'react';
 
 export default interface ShowCaseWidgetProps {
   widgetName: string;
   type: JSONSchema7TypeName;
 }
+
+export type VernaWidgetProps = Partial<WidgetProps> & { onChange: ChangeEventHandler<unknown> };
```

Comment:
Just for homogeneity to define Props type we could use `interface` or `type` but if possible not both.

```suggestion
export interface VernaWidgetProps extends Partial<WidgetProps> {
  onChange: WidgetProps['onChange'] & ChangeEventHandler<unknown>;
};
```

Furthermore, just to be sure, do you really want the type of `onChange` is `WidgetProps['onChange'] & ChangeEventHandler<unknown>` or do you want that is equal to `ChangeEventHandler<unknown>` ?

Reply:
I guess I wanted the onchange of ChangeEventHandler<unknown> and not the one of widgetProps, but i now have a warning that event.target.value doesn't exist.. (in both cases, weird)

Reply:
I overloaded a onChange on the input template and it's good now